File: src/main/java/me/krit/hychat/server/chat/ChatContext.java
package me.krit.hychat.server.chat;

public class ChatContext
{
    public String title;
    private String chatPrefix;
    public ChatType type;

    public ChatContext(String title, String chatPrefix, ChatType type)
    {
        this.title = title;
        this.chatPrefix = chatPrefix;
        this.type = type;
    }

    public boolean messageQualifiesForContext(String message)
    {
        return message.startsWith(chatPrefix);
    }
}

File: src/main/java/me/krit/hychat/server/chat/ChatContext.kt
package me.krit.hychat.server.chat;

public class ChatContext
{
    public String title;
    public String chatPrefix;
    public ChatType type;
    public String sendToContextCommand;

    public ChatContext(String title, String chatPrefix, ChatType type, String sendToContextCommand)
    {
        this.title = title;
        this.chatPrefix = chatPrefix;
        this.type = type;
        this.sendToContextCommand = sendToContextCommand;
    }

    public boolean messageQualifiesForContext(String message)
    {
        System.out.println(message);
        return message.startsWith(chatPrefix);
    }
}
